kite-data/kite-data-core/src/test/java/org/kitesdk/data/spi/predicates/TestInToFromString.java

```diff
@@ -117,6 +127,16 @@ public void testMultipleIntegerValues() {
   }
 
   @Test
+  public void testMultipleIntegerValuesNormalized() {
+    // normalized predicate normalizes the string values,
+    // so they won't be sorted numerically.
+    // ensures that cases where the type may not be comparable directly
+    // that they will be consistently sorted
+    Assert.assertEquals("In#toString(INT)",
+        "123,45", Predicates.in(45,123).toNormalizedString(INT));
```

Comment:
I think this would be possible, although a little awkward (different behaviors in the switch for the different Schema types). I think it's worth it to leave it as just lexicographic in all cases. It isn't as human friendly but it is highly consistent.